app/stores/AppStore.js

```diff
@@ -96,21 +109,23 @@ export default class AppStore extends Store {
     this.getTermsOfUseAcceptanceRequest.execute();
   };
 
+  _getSendLogsChoice = async () => {
+    const sendLogs = await this.getSendLogsChoiceRequest.execute().promise;
+    ipcRenderer.send('send-logs-choice', sendLogs);
```

Comment:
Hmm … why are you `send`ing something to the main process in a getter function? This is a bit confusing for me.